polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -1149,19 +1143,27 @@ export class CodeCell extends Cell {
      *          So, width data goes from cell -> editor.
      *          A horizontal scrollbar is necessary if the text content overflows.
      */
-    layout() {
+    private previousHeight: number = 0;
+    layout(forceWidth?: number) {
+        if (!this.el.parentNode) {
+            return;
+        }
         const editorLayout = this.editor.getLayoutInfo();
         // set the height to the height of the text content. If there's a scrollbar, give it some room so it doesn't cover any text
         const lineHeight = this.editor.getOption(editor.EditorOption.lineHeight);
         // if the editor height is less than one line we need to initialize the height with both the content height as well as the horizontal scrollbar height.
         const shouldAddScrollbarHeight = editorLayout.height < lineHeight;
         const height = this.editor.getContentHeight() + (shouldAddScrollbarHeight ? editorLayout.horizontalScrollbarHeight : 0);
+        if (this.previousHeight === height) {
+            return;
+        }
+        this.previousHeight = height;
         // the editor width is determined by the container's width.
-        const width = this.editorEl.clientWidth;
+        const width = forceWidth ?? Main.get.splitView.centerWidth - 66;
+        console.log('layout cell', width, this.id);
```

Comment:
console log

Reply:
👍  will remove before this PR goes anywhere